packages/process/src/node/process.ts

```diff
@@ -108,6 +108,9 @@ export abstract class Process {
         return this.errorEmitter.event;
     }
 
+    abstract onData(listener: (buffer: string) => void): void;
```

Comment:
please replace with Theia event API

Reply:
i will take advantage of the `outputStream` Paul added to `abstract class Process` in https://github.com/theia-ide/theia/pull/4586. so i will leave this function as is for now, and rewrite it after PR 4586 gets merged.

Reply:
ok i removed this function and used `outputStream.on()` instead